pipeline.jsonc

```diff
@@ -1,8 +1,8 @@
 {
     "eosio": {
         "environment": {
-            "IMAGE_TAG": "1.8.0-rc2",
-            "HELPER_ENABLED": true
+            "HELPER_ENABLED": true,
+            "IMAGE_TAG_VERSION": "v1"
```

Comment:
Same as above, this should be deleted here and implemented in `auto-buildkite-pipelines` unless `IMAGE_TAG_VERSION` is overriding some default behavior.

Reply:
It is...